### PR TITLE
Downloads desktop

### PR DESCRIPTION
**Pull request by corwintines**
Desktop styling for downloads page

### CODE REVIEW

File: src/components/UI/downloads/SpecificVersionsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { FC } from 'react';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const SpecificVersionsSection: FC<Props> = ({ children }) => {
+  return (
+    <Flex
+      id='specificversions'
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/components/UI/downloads/SpecificVersionsSection.tsx:11]: I can remove, just needed for pgpsignatures

File: src/pages/downloads.tsx [hunk 2]
```diff
@@ -92,73 +92,115 @@ const DownloadsPage: NextPage = () => {
...
+                  </Text>
+                </Link>
+              </Stack>
+            </Flex>
           </DownloadsSection>
 
-          <DownloadsSection sectionTitle='OpenPGP Signatures' id='pgpsignatures'>
-            <Stack p={4} borderBottom='2px solid' borderColor='brand.light.primary'>
+          <DownloadsSection
+            id='pgpsignatures'
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/pages/downloads.tsx:196]: Yes, in the specific versions section on the downloads page there is a link to this.

File: src/components/UI/homepage/HomeHero.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const HomeHero: FC = () => {
           textStyle='h1'
           mb={{ base: 2, md: 4 }}
           textAlign={{ base: 'center', md: 'left' }}
-          fontSize={{ base: '3rem', md: '6rem'}}
+          fontSize={{ base: '3rem', md: '6rem' }}
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/components/UI/homepage/HomeHero.tsx:22]: Ah thanks!

File: src/components/UI/homepage/HomeHero.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const HomeHero: FC = () => {
...
           lineHeight={{ md: '6rem' }}
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/components/UI/homepage/HomeHero.tsx:23]: Thank you!

File: src/components/UI/downloads/DownloadsHero.tsx [hunk 5]
```diff
@@ -29,61 +40,92 @@ export const DownloadsHero: FC<DownloadsHero> = ({
...
-        <Image src='/images/pages/gopher-downloads-front-light.svg' alt='Gopher plugged in' />
-      </Stack>
+    <Flex border='3px solid' borderColor='brand.light.primary' py={4} px={4} flexDirection='row'>
+      <Flex flexDirection='column'>
+        <Flex mb={4} direction={{ base: 'column', md: 'row-reverse' }}>
+          <Flex
+            py={{ base: 0 }}
+            px={{ base: 8 }}
+            flex={{ base: 'none' }}
+            display={{ base: '', lg: 'none' }}
```
[the remark this answers is not in the document]
Comment (corwintines) [reply src/components/UI/downloads/DownloadsHero.tsx:50]: Meant to come back to that, thanks for catching

File: src/components/UI/downloads/DownloadsHero.tsx [hunk 6]
```diff
@@ -29,61 +39,100 @@ export const DownloadsHero: FC<DownloadsHero> = ({
...
+                  <HStack spacing={4}>
+                    <Stack alignItems='center'>
+                      <Image
+                        src={DOWNLOAD_HEADER_BUTTONS[key].image}
+                        alt={DOWNLOAD_HEADER_BUTTONS[key].imageAlt}
+                      />
+                    </Stack>
+                    <Box>
+                      <Text textStyle='downloads-button-label'>
+                        For {DOWNLOAD_HEADER_BUTTONS[key].name}
```
Comment (nhsz) [inline src/components/UI/downloads/DownloadsHero.tsx:109]: Font sizes are different on design (https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1043%3A8883&t=BRNWeIoFi4HUMoPH-0) with platform name bigger (20px on desktop) and build name smaller (14px). They're different on mobile too (16px and 14px).

**Review by nhsz — APPROVED** [review]
Niceeee, lgtm! 🙌

**Pull request merged**